datalad_dataverse/remote.py

```diff
@@ -34,16 +35,26 @@ def initremote(self):
             raise RuntimeError("Cannot find dataset")
 
     def prepare(self):
-        raise
         pass
 
     def checkpresent(self, key):
-        raise
-        pass
+        api = NativeApi(self.annex.getconfig('url'), os.environ.get('DATAVERSE_API_TOKEN', None))
```

Comment:
This should eventually go into `prepare` and be available as `self.api`, I guess. However, just a remark, doesn't need to be in this pull requests.